tests/EnumConverter.UnitTests/EnumConverterTest.cs

```diff
@@ -8,186 +8,120 @@ namespace EnumConverterLibrary.UnitTests
 {
     public class EnumConverterTest
     {
-        [Fact]
-        public void ToAnother_CanConvertInputEnumToAnotherEnum_AnotherEnum()
+        [Theory]
+        [InlineData (InputEnum.First, AnotherEnum.First)]
+        [InlineData (InputEnum.Second, AnotherEnum.Second)]
+        [InlineData (InputEnum.Third, AnotherEnum.Third)]
+        public void ToAnother_CanConvertInputEnumToAnotherEnum_AnotherEnum(
+            InputEnum inputEnum, 
+            AnotherEnum expectedEnum)
         {
-            // Arrange.
-            InputEnum inputEnumFirst = InputEnum.First;
-            InputEnum inputEnumSecond = InputEnum.Second;
-            InputEnum inputEnumThird = InputEnum.Third;
-
             // Act.
-            AnotherEnum anotherEnumFirst = inputEnumFirst.ToAnother<AnotherEnum>();
-            AnotherEnum anotherEnumSecond = inputEnumSecond.ToAnother<AnotherEnum>();
-            AnotherEnum anotherEnumThird = inputEnumThird.ToAnother<AnotherEnum>();
+            AnotherEnum anotherEnum = inputEnum.ToAnother<AnotherEnum>();
 
             // Assert.
-            Assert.IsType<AnotherEnum>(anotherEnumFirst);
-            Assert.IsType<AnotherEnum>(anotherEnumSecond);
-            Assert.IsType<AnotherEnum>(anotherEnumThird);
-
-            Assert.Equal(AnotherEnum.First, anotherEnumFirst);
-            Assert.Equal(AnotherEnum.Second, anotherEnumSecond);
-            Assert.Equal(AnotherEnum.Third, anotherEnumThird);
+            Assert.IsType<AnotherEnum>(anotherEnum);
+            Assert.Equal (expectedEnum, anotherEnum);
         }
 
-        [Fact]
-        public void ToAnother_CanConvertInputEnumToAnotherEnumWithIgnoreCase_AnotherEnum()
+        [Theory]
+        [InlineData (InputEnumLowerCase.first, InputEnumUpperCase.FIRST, AnotherEnum.First)]
+        [InlineData (InputEnumLowerCase.second, InputEnumUpperCase.SECOND, AnotherEnum.Second)]
+        [InlineData (InputEnumLowerCase.third, InputEnumUpperCase.THIRD, AnotherEnum.Third)]
+        public void ToAnother_CanConvertInputEnumToAnotherEnumWithIgnoreCase_AnotherEnum(
+            InputEnumLowerCase inputEnumLowerCase, 
+            InputEnumUpperCase inputEnumUpperCase, 
```

Comment:
We can also delete this parameter and add more InlineData
```cs
        [InlineData(InputEnumLowerCase.first, AnotherEnum.First)]
        [InlineData(InputEnumLowerCase.second, AnotherEnum.Second)]
        [InlineData(InputEnumLowerCase.third, AnotherEnum.Third)]
        [InlineData(InputEnumUpperCase.FIRST, AnotherEnum.First)]
        [InlineData(InputEnumUpperCase.SECOND, AnotherEnum.Second)]
        [InlineData(InputEnumUpperCase.THIRD, AnotherEnum.Third)]
```
Oh, and don't forget to change parameter name to `inputEnum` instead of `inputEnumLowerCase`

Reply:
How do I create a method that excepts two different types of enums (InputEnumLowerCase and InputEnumUpperCase) into one parameter?

Reply:
I didn't see that it are two different types, sorry :)